source/javascripts/frob_core_helpers.js

```diff
@@ -327,13 +327,22 @@
      * @param {Number} [offset=0] - Additional offset to add to the scrollTop
      */
     smoothScroll: function($target, duration, delay, offset){
+      var _this = this;
+
       duration = this.setDefault(duration, 2000);
       delay = this.setDefault(delay, 100);
       offset = this.setDefault(offset, 0);
 
       setTimeout(function(){
+        var targetOffset = $target.offset().top + offset;
```

Comment:
Variables are snake_case, functions are camelCase. Sorry, I'm just dogging you with comments today.